src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -12,12 +12,19 @@
 from yandex_cloud_ml_sdk._types.domain import BaseDomain
 from yandex_cloud_ml_sdk._types.expiration import ExpirationConfig, ExpirationPolicyAlias
 from yandex_cloud_ml_sdk._types.misc import UNDEFINED, PathLike, UndefinedOr, coerce_path, get_defined_value, is_defined
+from yandex_cloud_ml_sdk._utils.doc import doc_from
 from yandex_cloud_ml_sdk._utils.sync import run_sync, run_sync_generator
 
 from .file import AsyncFile, File, FileTypeT
 
 
 class BaseFiles(BaseDomain, Generic[FileTypeT]):
+    """Files domain, which contains API for working with files.
+    Files is a part of `Assistants API`_, which is the only place you could use it.
+    Provides upload, get and list methods that allow you to work with remote file objects you created earlier.
+
+    .. _Assistants API: https://yandex.cloud/ru/docs/foundation-models/assistants/api-ref/Assistant/
```

Comment:
When I talking about "assistants api" in this doc, I mean 
`sdk.assistants`, `sdk.files`, `sdk.search_indexes`, etc.
I want to say that this single product.
Maybe there will be distinct page in doc about Assistants, and it would be good ref from here.

And we don't want to show our user any protobufs, so this link need to be removecd.